src/build/ChocolateyTemplates/gittfs.nuspec

```diff
@@ -21,7 +21,7 @@ ${ReleaseNotesContents}
     <licenseUrl>https://github.com/git-tfs/git-tfs/blob/master/LICENSE</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <dependencies>
-      <dependency id="git" version="1.9.5.20150320" />
```

Comment:
This means at least this version, but pick the latest available

Reply:
@bdukes ` version="1.9.5.20150320"` means `>=1.9.5.20150320`, where  `version="[1.9.5.20150320]"` means `=1.9.5.20150320` (exactly and only this version)

Reply:
Here's my install log, where it installed _git.install 1.9.5.20150320_ during `choco upgrade all -y`, as part of upgrading `gittfs`: https://gist.github.com/bdukes/8c8ce780e280b2ea8ae55d899dcd1b2f#file-choco-summary-log-L874-L883, if something was weird on my machine that caused that, feel free to disregard

Reply:
You know, we did move up to a newer NuGet version awhile back. I wonder if they snuck in another place where it was looking for lowest dependency. Our integration tests should be looking at that.

When I ran this on a test machine, it went for the latest version of git.

Reply:
@bdukes the nuget log has a bug in it:

~~~
2017-10-16 16:35:00,530 1680 [INFO ] - [NuGet] Attempting to resolve dependency 'git (= 1.9.5.20150320)'.
2017-10-16 16:35:01,249 1680 [INFO ] - [NuGet] Attempting to resolve dependency 'git.install (= 2.14.2.20170927)'.
2017-10-16 16:35:14,280 1680 [INFO ] - [NuGet] Attempting to resolve dependency 'chocolatey-core.extension (= 1.3.1)'.
2017-10-16 16:35:14,296 1680 [INFO ] - [NuGet] Attempting to resolve dependency 'chocolatey (= 0.10.7)'.
2017-10-16 16:35:14,983 1680 [INFO ] - [NuGet] Installing 'git.install 2.14.2.20170927'.
~~~

It looks like it is going for an exact version, but that it not what it resolves for me.

Reply:
@bdukes you might file a support ticket and provide the chocolatey.log file so our team can take a closer look.